CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [v1.4.3](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.4.3) (2022-07-12)
+## Fixed
+
+* INTMDB-335: Add option for multiple weekly monthly schedules @martinstibbe in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/784
+* INTMDB-348: autoexport parameter not being set via provider https://jira.mongodb.org/browse/INTMDB-348
```

Comment:
The jira link can't be seen externally so let's leave https://jira.mongodb.org/browse/INTMDB-348 - we only use those in the PR for internal tracking.   I'd just point to the same PR as 335 since they are combined into one. 

Please also make sure to update this with the other items in the release ticket

Reply:
Updated Changelog to include additional tickets